### PR TITLE
KUBE-364: added node_group_arn

### PR DESCRIPTION
**Pull request by cirisked**

### CODE REVIEW

File: castai/resource_node_configuration.go [hunk 1]
```diff
@@ -286,6 +287,12 @@ func resourceNodeConfiguration() *schema.Resource {
 								},
 							},
 						},
+						"node_group_arn": {
```
Comment (Tsonov) [inline castai/resource_node_configuration.go:290]: Add a const for the field name (like the others); makes it easier to list all fields at the top

File: castai/resource_node_configuration.go [hunk 2]
```diff
@@ -286,6 +287,12 @@ func resourceNodeConfiguration() *schema.Resource {
...
+							Type:             schema.TypeString,
+							Optional:         true,
+							Description:      "Cluster's node group ARN used for CAST provisioned node pools",
```
Comment (Tsonov) [inline castai/resource_node_configuration.go:293]: Is this only used for hibernate or for all CAST pools? Does it apply only to EKS?  Some additional context in the description would be useful.

Comment (cirisked) [reply castai/resource_node_configuration.go:293]: EKS only, will update description.

File: castai/resource_node_configuration.go (hunk 1 above)
Comment (cirisked) [reply castai/resource_node_configuration.go]: done

File: castai/resource_node_configuration.go [hunk 3]
```diff
@@ -178,6 +179,13 @@ func resourceNodeConfiguration() *schema.Resource {
 							},
 							Description: "Cluster's security groups configuration for CAST provisioned nodes",
 						},
+						"node_group_arn": {
```
Comment (aldor007) [inline castai/resource_node_configuration.go:182]: Can you also add acceptance test case?

Comment (cirisked) [reply castai/resource_node_configuration.go:182]: done

**Review by aldor007 — APPROVED** [review]

**Pull request merged**